src/tickets/tickets.controller.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { Test } from '@nestjs/testing';
+import { PurchaseTicketDto } from './dto/purchase-ticket.dto';
+import { TicketsService } from './tickets.service';
+import { TotpService } from './totp.service';
+
+describe('TicketsController', () => {
```

Comment:
@MichaelSafwatHanna can you check this one, to meet the latest changes as the `user` module?